src/containers/SecretsModal/SecretsModal.js

```diff
@@ -59,15 +59,14 @@ export /* istanbul ignore next */ class SecretsModal extends Component {
       serviceAccount: '',
       annotations: [
         {
-          label: `tekton.dev/git-0`,
+          key: 'tekton.dev/git-0',
```

Comment:
what's the difference between `key` & `keyPlaceholder`?

Reply:
The `key`, is the actual text that is in the "key" TextInput (the `value` prop), whereas the `keyPlaceholder` is the placeholder text in the "key" TextInput (the `placeholder` prop).

Do you think that these names are confusing/do you have an idea for an alternative name?